### ISSUE TITLE
Undefined index: graphql

### ISSUE THREAD

**Issue by beeldr** [codemonauts/craft-instagram-feed#53]
Hey guys,

I've been using your plugin for a while now and i thinks it great! Unfortunately when I activate the proxy with my personal key the website crashes (though the same proxy works fine on another of my websites).

Used config file
```
<?php

return [

    'proxyKey' => '<mykey>',
    'useProxy' => true,
    'useGuzzle' => true,
    'timeout' => 10,
];
```

Error message
```
2021-05-18 09:10:27 [-][1][bbvfier9udbrv39georro1al8k][error][yii\base\ErrorException:8] yii\base\ErrorException: Undefined index: graphql in /var/www/vhosts/example.com/httpdocs/vendor/codemonauts/craft-instagram-feed/src/services/InstagramService.php:162
Stack trace:
#0 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/ErrorHandler.php(76): yii\base\ErrorHandler->handleError()
#1 /var/www/vhosts/example.com/httpdocs/vendor/codemonauts/craft-instagram-feed/src/services/InstagramService.php(162): craft\web\ErrorHandler->handleError()
#2 /var/www/vhosts/example.com/httpdocs/vendor/codemonauts/craft-instagram-feed/src/services/InstagramService.php(51): codemonauts\instagramfeed\services\InstagramService->getInstagramTagData()
#3 /var/www/vhosts/example.com/httpdocs/vendor/codemonauts/craft-instagram-feed/src/variables/InstagramFeedVariable.php(11): codemonauts\instagramfeed\services\InstagramService->getFeed()
#4 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Extension/CoreExtension.php(1511): codemonauts\instagramfeed\variables\InstagramFeedVariable->getFeed()
#5 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute()
#6 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/54/541ed543f52e4e8980fd48baa5361cba49c8e88ead0f76dd07413fd075cbbbf9.php(41): craft\helpers\Template::attribute()
#7 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_f2144b4525d7f58aff99c533d58dec700be86ff33c09b9634a4d3a3f385d586a->doDisplay()
#8 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#9 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/96/9674ab6b333298af86ac09a32e99f1b7a34075ce62df4a09c96274d2702633f9.php(69): Twig\Template->display()
#10 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_1a55f3bea29ec200124923e51477c528f9570da38eae62e3e8003194b7d4e0cf->doDisplay()
#11 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#12 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/c3/c3af3a3564b22141ab0f2d452f7b69f2bc75ef76ed04e5fbb3dc2fae5a2e47f4.php(46): Twig\Template->display()
#13 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_aa13b344613fbb25346ac2e8f8439f9a1696ffda5260cf619f50cbd61783c843->doDisplay()
#14 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#15 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/64/64f6a6fd8ac44890777cc79cf3b6b189233ba28ac04d47d95e14d56139297e95.php(124): Twig\Template->display()
#16 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_3b9c71a224c354b6ad5b975c3b2e3a51f2815a3f1e376a9f191a195913c72753->doDisplay()
#17 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#18 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/49/49045e82490a162883d118283166f16ba4eb10bf4fb05772a94796ce92119dc5.php(41): Twig\Template->display()
#19 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_4bf9f501be3f68dd84aebbc079a90366661776d93eef2abd265c96e94e470391->doDisplay()
#20 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#21 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/ba/bab6f32117369b81ab2c003d08cf9e60ec9be651be836b39f9272f702e7f3a10.php(56): Twig\Template->display()
#22 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(182): __TwigTemplate_503a5803bba0f4b6576a67d27e6f4c73ab30817fa2253cb7566d625538cfd608->block_page_main()
#23 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/73/7384d716fb4878ed222eef4d398a873cb3c2f53440ef2427ae9835aaa0a181e4.php(442): Twig\Template->displayBlock()
#24 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_3765b26491e15997bd74d394d909994f7c2ae9eba77187446d2b96c76050dba1->doDisplay()
#25 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#26 /var/www/vhosts/example.com/httpdocs/storage/runtime/compiled_templates/ba/bab6f32117369b81ab2c003d08cf9e60ec9be651be836b39f9272f702e7f3a10.php(43): Twig\Template->display()
#27 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(405): __TwigTemplate_503a5803bba0f4b6576a67d27e6f4c73ab30817fa2253cb7566d625538cfd608->doDisplay()
#28 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#29 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Template.php(390): Twig\Template->display()
#30 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render()
#31 /var/www/vhosts/example.com/httpdocs/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render()
#32 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/View.php(389): Twig\Environment->render()
#33 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/View.php(450): craft\web\View->renderTemplate()
#34 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/Controller.php(257): craft\web\View->renderPageTemplate()
#35 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/controllers/TemplatesController.php(100): craft\web\Controller->renderTemplate()
#36 [internal function]: craft\controllers\TemplatesController->actionRender()
#37 /var/www/vhosts/example.com/httpdocs/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#38 /var/www/vhosts/example.com/httpdocs/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams()
#39 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction()
#40 /var/www/vhosts/example.com/httpdocs/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction()
#41 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/Application.php(278): yii\base\Module->runAction()
#42 /var/www/vhosts/example.com/httpdocs/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction()
#43 /var/www/vhosts/example.com/httpdocs/vendor/craftcms/cms/src/web/Application.php(263): yii\web\Application->handleRequest()
#44 /var/www/vhosts/example.com/httpdocs/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest()
#45 /var/www/vhosts/example.com/httpdocs/web/index.php(21): yii\base\Application->run()
#46 {main}

```

**Comment by kringkaste** [comment]
Hi, could you tell us the hashtag you want to fetch?

**Comment by fleaz** [comment]
Hey @beeldr 
we just verified that this is a problem in our plugin, because Instagram changed the page structure of hashtag pages, therefore our plugin can't parse them anymore.
We try to fix this asap, so have an eye on the releases :)

**Comment by fleaz** [comment]
Hey,
we just released 1.1.3 which fixes the problem with accessing hashtags :)
